python/datawrangling/cms-openpayment/general_payment_process_manufacturer.py

```diff
@@ -0,0 +1,112 @@
+"""
+This script attempts to create a canonical name for the names of the manufacturers 
+and GPOs. The script first checks whether the mapping between ID and name/state/country
+information is consistent, then converts the names into a canonical form and outputs
+a csv containing this information.
+
+Inconsistency in raw data can be handled by specifying a yaml file as the dict 
+parameter. This yaml file should contain a sequence of dictionaries, each dictionary
+containing at least the id key. Information in this file is written directly to the 
+output csv. 
+
+author: sangxia
+"""
+
+import pandas as pd
+import unicodedata
+import numpy as np
+import csv
+import argparse
+import yaml
+import re
+import copy
+
+def name_canon(s):
+    ret = unicodedata.normalize('NFKD',s.strip()).encode('ascii','ignore').decode()
```

Comment:
Ah, nevermind. I see pandas seems to use the same `unicodedata` calls. I'm all for consistency in this case. 👍